integration_tests/commands/async/json_test.go

```diff
@@ -1095,7 +1094,7 @@ func TestJsonObjLen(t *testing.T) {
 				cmd := tcase.commands[i]
 				out := tcase.expected[i]
 				result := FireCommand(conn, cmd)
-				assert.DeepEqual(t, out, result)
+				assert.Equal(t, out, result)
```

Comment:
This might be failing test cases

Reply:
Hmm, okay I will take a look at it now

Reply:
Running the test in isolation, it works, but running with others makes it fail.

![json_test_fail](https://github.com/user-attachments/assets/8a58cefe-9210-4dba-8767-dab639d3d501)


I also noticed all tests in ZCOUNT passes but at the end it shows FAIL
![zcount_fail](https://github.com/user-attachments/assets/b7b455b6-cf4b-4895-b9f6-34ff1d2a5749)


Any thoughts on this?

Reply:
The Fail could be from different test case that ran earlier. Can you please search for Fail and see if there are logs for this

Reply:
FAIL below the ZCOUNT is also from json_test.  
JSON.OBJKEYS with empty path TC in websocket is failing.

Reply:
Do we have a fix for this one @c-harish ?

Reply:
@apoorvyadav1111. Assert failing TC with arraysArePermutations instead of equal. JSON.OBJKEYS orders are not deterministic.

Reply:
Okay I will do this now. Thanks

Reply:
Hi, @tdadadavid. Please user ElementsMatch for assertion of all json.objkeys testcases before merging. Thanks.

Reply:
Thanks @c-harish, @tdadadavid You can also use elementsMatch https://pkg.go.dev/github.com/stretchr/testify/assert#ElementsMatch